ArduCopter/motors.cpp

```diff
@@ -6,73 +6,6 @@
 
 static uint32_t auto_disarm_begin;
 
-// arm_motors_check - checks for pilot input to arm or disarm the copter
-// called at 10hz
-void Copter::arm_motors_check()
-{
-    static int16_t arming_counter;
-
-    // check if arming/disarm using rudder is allowed
-    AP_Arming::RudderArming arming_rudder = arming.get_rudder_arming_type();
-    if (arming_rudder == AP_Arming::RudderArming::IS_DISABLED) {
-        arming_counter = 0;
-        return;
-    }
-
-#if TOY_MODE_ENABLED
-    if (g2.toy_mode.enabled()) {
-        // not armed with sticks in toy mode
-        return;
-    }
-#endif
-
-    // ensure throttle is down
-    if (channel_throttle->get_control_in() > 0) {
```

Comment:
this check is lost I think, in master we won't get a failed arm event, in this PR I think we will

Reply:
Yep, as we do in Plane, so we become consistent.  Something to discuss!